detectors/detector.py

```diff
@@ -12,6 +12,8 @@ def get_bounding_boxes(frame, model):
         from detectors.haarcascade import get_bounding_boxes as gbb
     elif model == 'tfoda':
         from detectors.tfoda import get_bounding_boxes as gbb
+    elif model == 'detectron2':
```

Comment:
Can you provide a sample model with which to test this detector?

Reply:
Cityscapes or Pascal VOC seems to give good results: https://github.com/facebookresearch/detectron2/blob/master/MODEL_ZOO.md#cityscapes--pascal-voc-baselines

Reply:
Thanks @luiscosio